sky/provision/kubernetes/utils.py

```diff
@@ -389,7 +394,7 @@ def get_gke_accelerator_name(accelerator: str) -> str:
     elif accelerator == 'H200':
         # H200s on GCP use this label format
         return 'nvidia-h200-141gb'
-    elif accelerator.startswith('tpu-'):
+    elif accelerator.startswith('amd-'):
         return accelerator
```

Comment:
```suggestion
    elif accelerator.startswith('tpu-'):
        return accelerator
    elif accelerator.startswith('amd-'):
        return accelerator
```
so the tpu support is not disrupted by this PR